docs/what-is-bitcoin.md

```diff
@@ -0,0 +1,71 @@
+## Overview
+
+Bitcoin is a peer-to-peer digital currency and network. Bitcoins can be sent and received through
```

Comment:
LINE 3: Change "network" to "cryptographically secure network" The general public needs to feel secure in The Bitcoin Network with their money.

Reply:
LINE 3: Change to "Bitcoins can be sent and received using a Bitcoin wallet using"